ReadingList/ViewControllers/Edit/EditBookReadState.swift

```diff
@@ -99,6 +99,15 @@ class EditBookReadState: FormViewController {
         monitorThemeSetting()
     }
 
+  override func viewDidAppear(_ animated: Bool) {
+    super.viewDidAppear(animated)
+    if self.book.readState == .reading {
```

Comment:
```suggestion
    // If we are editing a book (not adding one), pre-select the current page field
    if self.book.readState == .reading && self.book.changedValues().isEmpty {
```
I think we shouldn't pre-select the field arriving on this page when adding a new book. To do that, I added this extra check. (If this page has been brought up from an "Edit" tap, the book will have no changedValues; if this is a new book in the process of being created, there will be numerous changedValues.)

Reply:
Good find. Didn't thought that when I implemented this.